Footer

D3TRU04 opened the pull request:
<img width="1474" alt="Screenshot 2024-06-03 at 8 10 21 PM" src="https://github.com/Texas-Capital-Collective/tcc-golf/assets/68484345/56920198-a52e-4ec8-89ad-6a09f4f09dd8">

D3TRU04 committed:
Updates

txst54 commented:
1. Good work. Logo at the footer should be vertically aligned with the logo in the navbar. Add in the icons from the figma. You should try using a library like [phosphor](https://phosphoricons.com/). They have a react library that you can [install](https://github.com/phosphor-icons/react). Padding and spacing should be increased to match the figma as closely as possible.

D3TRU04 committed:
Updates

Can't figure out how to vertically align with footer logo with navbar logo. Even chatgpt was wrong.

D3TRU04 committed:
Updates

D3TRU04 committed:
Updates

txst54 reviewed (approved):
awesome

D3TRU04 committed:
Update yarn.lock

D3TRU04 committed:
Merge branch 'master' into footer
